__init__.py

```diff
@@ -251,14 +254,44 @@ def match(self):
 
     def remove(self):
         '''remove()
-
-        From HDF5 file remove group paired with this variable.
-
+        Removes associated HDF5 group.
+        Method called on specified variable in the workspace. 
+        
+        Examples
+        ---------
+        Add example variable (named "E") to the HDF5 file with index = 0 in the workspace::
```

Comment:
Please change naming convention for variables. var1, var2, ... seems to be fine.
Also default index = -1 would be better.